.github/workflows/docker.yaml

```diff
@@ -0,0 +1,47 @@
+name: Publish to DockerHub
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - 'v*'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: marshallasch/dugong
```

Comment:
this image name should be changes to be under a namespace you own (I have not actually published a version of the image from my fork)